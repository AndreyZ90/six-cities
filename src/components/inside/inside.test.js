import React from 'react';
import TestRenderer from 'react-test-renderer';

import Inside from '@/components/inside/inside';

const childrenNode = <div className="some-children-node">Inside children node</div>;

describe(`Inside component snapshot`, () => {
  test(`Should correctly render Inside component`, () => {
    const tree = TestRenderer.create(
        <Inside>
          {childrenNode}
        </Inside>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
