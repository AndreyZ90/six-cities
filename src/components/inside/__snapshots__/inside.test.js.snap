// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inside component snapshot Should correctly render Inside component 1`] = `
<div
  className="property__inside"
>
  <h2
    className="property__inside-title"
  >
    What's inside
  </h2>
  <div
    className="some-children-node"
  >
    Inside children node
  </div>
</div>
`;
